docs/installationguide/ubuntu.rst

```diff
@@ -3,8 +3,64 @@ Ubuntu
 
 .. contents::
 
-Easy Way: Ubuntu 18.04 (Bionic)
--------------------------------
+Ubuntu 22.04 (Jammy)
+--------------------
+These instructions are for a brand new ubuntu 22.04 LTS system which does not have ZM installed.
+
+**Step 1:** Update system
+
+::
+
+    sudo apt-get update && sudo apt upgrade -y
+
+**Step 2:** Update repositories (Optional)
+
+ZoneMinder is part of the current standard Ubuntu repository, but sometimes the official repository can lag behind.
+To find out check our `releases page <https://github.com/ZoneMinder/zoneminder/releases>`_ for the latest release.
+    
+Alternatively, the ZoneMinder project team maintains a `PPA <https://askubuntu.com/questions/4983/what-are-ppas-and-how-do-i-use-them>`_, which is updated immediately following a new release of ZoneMinder.
+To use this repository instead of the official Ubuntu repository, enter the following commands:
+
+::
+
+        sudo apt install -y software-properties-common
+        sudo add-apt-repository ppa:iconnor/zoneminder-1.36
+        sudo apt update
+
+**Step 3:** Install Zoneminder
+
+::
+        sudo apt install -y zoneminder
+
+**Step 4:** Set permissions
+
+::
+
+        sudo chown -R www-data:www-data /usr/share/zoneminder/
```

Comment:
Why are you doing this?  This is bad for security and is not needed. Otherwise these updates are great.

Reply:
Well observed!

I re-tested all the steps without running this command and it's works fine, so yes better to remove it.